strawberry_django/filters.py

```diff
@@ -167,6 +169,13 @@ def get_filters(self):
             return type_._django_type.filters
         return None
 
-    def get_queryset(self, queryset, info, pk=UNSET, filters=UNSET, **kwargs):
-        queryset = super().get_queryset(queryset, info, **kwargs)
+    def _apply_filters(
```

Comment:
Since this is here for overriding, I would suggest setting it as a public method to avoid someone in the future thinking it can be removed without breaking anything

Reply:
I was thinking the same but wasn't sure